### ISSUE TITLE
Cleanup orika-core pom.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [vadbch/orika#36]
```
There are a couple of dependencies in the orika-core pom.xml that should not be 
there:

1) Test dependencies in compile scope (instead of test scope):
* junit
* easymock
* spring-test

2) Useless dependencies in compile scope
* dozer
* spring
* h2database

3) Dependency on slf4j-simple instead of slf4j-api

This pushes these dependencies onto the project using orika. Please clean up 
your pom. Thanks :)
```

Original issue reported on code.google.com by `lighteater` on 30 Jul 2012 at 12:50

**Comment by GoogleCodeExporter** [comment]
```
Thanks
```

Original comment by `elaat...@gmail.com` on 30 Jul 2012 at 2:07
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `elaat...@gmail.com` on 30 Jul 2012 at 2:07

**Comment by GoogleCodeExporter** [comment]
```
Fixed in release 1.1.9
```

Original comment by `elaat...@gmail.com` on 25 Aug 2012 at 9:21
- Changed state: **Fixed**
